fortran/testpar/multidsetrw.F90

```diff
@@ -0,0 +1,236 @@
+! * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+!   Copyright by The HDF Group.                                               *
+!   Copyright by the Board of Trustees of the University of Illinois.         *
```

Comment:
And here

Reply:
Jo is going to investigate what to do since I copied another test and modified it. Do I still need the copyright from the copied code? She is also going to investigate how the uiuc copyright is still in the old files, but new code has been added; how does that work in terms of the new code?

Reply:
If you are mainly copying boilerplate, then I would remove the UIUC copyright. Code to just open files, etc. is not going to change much regardless of whether or not it's copied or written de novo.